setup.py

```diff
@@ -14,11 +16,55 @@ def c_files_in(directory):
         paths.append(os.path.join(directory, f))
     return paths
 
+def send_html_entities(entities_file, outfh, seen_entities):
+    # Convert entity list from HTML5 spec JSON and send it to gperf
+    with open(entities_file) as entitiesfh:
+        for entity, entityinfo in sorted(json.load(entitiesfh).items()):
+            if not entity.endswith(';'):
+                continue
+            if entity in seen_entities:
+                continue
+            seen_entities.add(entity)
+            # Some sanity checks on the codepoints
+            assert len(entityinfo['codepoints']) <= 2
+            for bad_range in [xrange(0, 8), xrange(11, 12), xrange(14, 31),
```

Comment:
This one's my bad, when I wrote the comment with the `xrange`s I forgot the upper bound on `xrange` was exclusive. All those upper bounds should be increased by one.